sormas-app/app/src/main/java/de/symeda/sormas/app/settings/SettingsActivity.java

```diff
@@ -82,6 +82,8 @@ public boolean onOptionsItemSelected(MenuItem item) {
 			String serverUrl = getActiveFragment().getServerUrl();
 			ConfigProvider.setServerRestUrl(serverUrl);
 
+			getActiveFragment().saveLbdsDebugUrl();
```

Comment:
should be done the same way as we are doing it for the server rest url, to have it consistent:

```
String lbdsDebugUrl= getActiveFragment().getLbdsDebugUrl();
ConfigProvider.setServerLbdsDebugUrl(lbdsDebugUrl);
```

Reply:
done